lua/efmls-configs/formatters/smlfmt.lua

```diff
@@ -0,0 +1,13 @@
+-- Metadata
+-- languages: sml
+-- url: https://github.com/shwestrick/smlfmt
+
+local fs = require('efmls-configs.fs')
+
+local formatter = 'smlfmt'
+local command = string.format("%s --preview-only ${FILENAME} | sed '1d;$d'", fs.executable(formatter))
```

Comment:
Oh I see what you mean from your first comment.

We can just force it to write to file and not take the changes from stdin. In this case remove `sed` and just use `--force`

```sh
%s --force ${INPUT}
```